spec/config-spec.js

```diff
@@ -1840,4 +1835,86 @@ describe('Config', () => {
       expect(atom.config.get('do.ray')).toBe('me')
     })
   })
+
+  describe('project specific configs', () => {
```

Comment:
Let the minor nitpicks begin! I have a few wording suggestions:

* I think `project specific configs` should be `project-specific settings` (here as well as in later `it/describe` strings)
* Can you drop the 'should' from the test descriptions (e.g. `should get` becomes `gets`)